_includes/tutorials/multi-joins/ksql/markup/dev/print-topic.adoc

```diff
@@ -0,0 +1,13 @@
+Finally, let's see what's available on the underlying Kafka topic for the new stream. We can print that out easily.
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/multi-joins/ksql/code/tutorial-steps/dev/print-topic.sql %}</code></pre>
++++++
+
+Notice that the key for each message includes not just the enriched order data, but also the rowtime of the records. It should look something like this:
```

Comment:
Any reason why this was mentioned?  If so, please explain...otherwise it might appear out of place

Reply:
changed this in latest to discuss why the item id is the key of the resulting topic records